requirements.txt

```diff
@@ -2,7 +2,7 @@ Click
 rdflib
 sparqlwrapper
 requests
-connexion
+connexion[swagger-ui]
```

Comment:
I wanted to ask before, what is the meaning of [swagger-ui]? just out of curiosity